spec/fuzzy-finder-spec.coffee

```diff
@@ -404,6 +404,9 @@ describe 'FuzzyFinder', ->
           dispatchCommand('toggle-buffer-finder')
 
       describe "when the active pane has an item for the selected path", ->
+        beforeEach ->
+          atom.config.set("fuzzy-finder.searchAllPanes", false)
```

Comment:
No good reason - just unfamiliarity with the way these tests handle state. Removed.